config/routes.rb

```diff
@@ -59,6 +59,12 @@
     member do
       put :enable
       put :disable
+      # Create the path: copy_dirmon_entry GET    /dirmon_entries/:id/copy(.:format)
+      get :copy
+      # Create the path: replicate_dirmon_entry PUT    /dirmon_entries/:id/replicate(.:format) rocket_job_mission_control/dirmon_entries#replicate
+      # Create the path:                        PATCH  /dirmon_entries/:id/replicate(.:format) rocket_job_mission_control/dirmon_entries#replicate
+      put :replicate
+      patch :replicate
```

Comment:
Hello Silvana, what's the need to have 2 similar routes here? Also if we are creating a brand new resource why not use post (instead of put or patch which are normally used for updating existing resources)

Reply:
The above is my response.